tests/unit/test_unit.py

```diff
@@ -0,0 +1,159 @@
+        return json.loads(card)
+
+    def get_lists():
+        """
+        mock list data generated by running curl command
+        curl -X GET "https://api.trello.com/1/boards/+self.board+/lists/?key=&token="
+        """
+        lists = '''
+        [
+            {"id": "5fd503e2db852a6e5a2fddab", "name": "TODO", "closed": false, "pos": 65535,"softLimit": null, "idBoard": "5fd503d7e780f63e718bf593", "subscribed": false},
+            {"id": "5fd5044419980d052c5ba01a", "name": "DOING", "closed": false, "pos": 131071,"softLimit": null, "idBoard": "5fd503d7e780f63e718bf593", "subscribed": false},
+            {"id": "5fd50448c2dc7974d2abed4b", "name": "DONE", "closed": false, "pos": 196607,"softLimit": null, "idBoard": "5fd503d7e780f63e718bf593", "subscribed": false}
+        ]
+        '''
+        return json.loads(lists)
+
+
+@pytest.fixture
+def lists():
+    # setup()
+    mock_lists = mock_trello_api.get_lists()
+    lists = [List(k, i) for i, k in enumerate(mock_lists)]
+    yield lists
+
+
+def test_list_id(lists):
+    assert lists[0].id == "5fd503e2db852a6e5a2fddab"
+
+
+@pytest.fixture
+def card():
+    # setup()
+    mock_card = mock_trello_api.get_card()
+    mock_list = [List(k, i) for i, k in enumerate(mock_trello_api.get_lists())]
+    card = Card(mock_list, mock_card)
+    yield card
+
+
+@pytest.fixture
+def cards():
+    # setup()
+    mock_cards = mock_trello_api.get_cards()
+    mock_list = [List(k, i) for i, k in enumerate(mock_trello_api.get_lists())]
+    cards = [Card(mock_list, i) for i in mock_cards]
+    yield cards
+
+
+@pytest.fixture
+def done_card(cards):
+    mock_done_card = [i for i in cards if i.listName == "DONE"][0]
+    yield mock_done_card
+
+
+def test_card_id(card):
+    assert card.id == "5f835afcf6400f7c70f9597e"
+
+
+def test_card_name(card):
+    assert card.name == "New Test Task"
+
+
+def test_card_list_id(card):
+    assert card.listId == "5fd503e2db852a6e5a2fddab"
+
+
+def test_card_list_name(card):
+    assert card.listName == "TODO"
+
+
+def test_card_due(card):
+    assert card.due == "2020-10-30"
+
+
+def test_card_date_last_activity(card):
+    assert card.dateLastActivity == "2020-10-11"
+
+
+def test_done_card(done_card):
+    assert done_card.listName == "DONE"
+
+
+class MockViewModel(ViewModel):
+    def get_date(self):
+        return date.fromisoformat("2020-12-30").strftime("%Y-%m-%d")
+
+
+@pytest.fixture
+def card_view_model(cards, lists):
+    yield MockViewModel(cards, lists)
+
+
+def test_view_model_get_date_mock(card_view_model):
```

Comment:
This unit test only tests the mock view model, so we aren't able to be confident that it actually says anything about the ViewModel object. For the unit tests, we really want to test against the object itself directly